docs/guides/user-containers/readme.md

```diff
@@ -262,16 +262,106 @@ allowed per workflow.
     GFS2 filesystems have special considerations since the mount directory contains directories for
     every compute node. See [GFS2 Index Mounts](#gfs2-index-mounts) for more info.
 
+### Targeting Nodes
+
+For container directives, compute nodes must be assigned to the workflow. The NNF software will
+trace the compute nodes back to their local NNF nodes and the containers will be executed on those
+NNF nodes. The act of assigning compute nodes to your container workflow instructs the NNF software
+to select the NNF nodes that run the containers.
+
+For the `jobdw` directive that is included above, the servers (i.e. NNF nodes) must also be assigned
+along with the computes.
+
 ## Running a Container Workflow
 
-TODO: Once the container exit strategy is complete, come back to this section to detail:
+Once the workflow is created, the WLM progresses it through the following states. This is a quick
+overview of the container-related behavior that occurs:
+
+- Proposal: Verify storages are provided according to the container profile.
```

Comment:
Can "storages" contain a hyperlink up to the "Container Storages" section?  I still stumble on some of the short-hand lingo.

Reply:
Good call